packages/ra-core/src/reducer/admin/notifications.spec.ts

```diff
@@ -8,11 +8,12 @@ import reducer from './notifications';
 
 describe('notifications reducer', () => {
     it('should return empty notification by default', () => {
-        assert.deepEqual([], reducer(undefined, { type: 'foo' }));
+        expect([]).toEqual(reducer(undefined, { type: 'foo' }));
```

Comment:
I suggest we swap the two operands here, I'm not a fan of this type of assertion where the result is put first.

Reply:
Done!